### ISSUE TITLE
PHP 5 or 7

### ISSUE THREAD

**Issue by ghost** [JMSDOnline/vstacklet#83]
I ran the installer and selected PHP 5 and PHP 7 was installed.

```
1) php7.0
2) php5
What version of php do you want? (Default php7.0): 2

Installing and Adjusting php5-fpm w/ OPCode Cache ... [ DONE ]

Do you want to install IonCube Loader? (Y/n):
Installing IonCube Loader ... vstacklet/vstacklet-ubuntu-stack.sh: line 597: /etc/php5/fpm/conf.d/20-ioncube.ini: No such file or directory
vstacklet/vstacklet-ubuntu-stack.sh: line 598: /etc/php5/fpm/php.ini: No such file or directory
[ DONE ]
```

```
root@testbox:~# php -v
PHP 7.0.6-1+donate.sury.org~xenial+1 (cli) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2016, by Zend Technologies
```

Thanks

**Comment by JMSDOnline** [comment]
Yes, follow along on the development branch. I am currently sorting this out as we speak.

### Edit

It appears I am asleep at the wheel since installing php7 as all my clients are being hosted on 7 with manual mysql management.. thus, I overlooked past 5 function. I am using the ondrej ppa and it is signed and needs to be expressed as php5.6 and is no longer php5. I am sorting this out in the script at this moment.

**Comment by ghost** [comment]
Ok, thanks. I am heading out, but will check back later tonight or tomorrow.

Thanks

**Comment by JMSDOnline** [comment]
Wonderful, thanks for offering your head on this. In my haste to migrate clients to php 7 I didn't really test this per public requirements.

### EDIT

PHP 5.6 and PHP 7.0 functions should now be resolved. Tested on 14.04 - 16.04. passing w/ a version bump.

[![Version 3.0.2-production](https://img.shields.io/badge/version-3.0.2-674172.svg?style=flat-square)](https://jmsolodesigns.com/code-projects/vstacklet/varnish-lemp-stack)

**Comment by ghost** [comment]
I just ran the dev installer and this appears to be fixed.

Thank you

**Comment by JMSDOnline** [comment]
@jonnyboy, that's good to hear, I did however, run into an issue with phpmyadmin which I am addressing now.

### EDIT

Issue fixed for php -v 5.6 - missing gettext dependency added on install #88
